README.md

```diff
@@ -13,6 +13,25 @@ This repository enables building resin.io for various devices.
 * docker
 * jq
 
+## Versioning
+
+`meta-resin` version is kept in `DISTRO_VERSION` variable. `resin-<board>` version is kept in the file called VERSION located in the root of the `resin-<board>` repository and read in the build as variable HOSTOS_VERSION.
+
+There were two version schemes in meta-resin. One that was used before v1.2 and one that is used from 1.2 as it follows:
+
+* Before v1.2
+    * The version of `meta-resin` is in the format of 3 numbers separated by dots.
+    * The version of `resin-<board>` has the same value as the version of `meta-resin` on which it is based on.
+    * Ex. For `meta-resin` release 1.1.1 there can be one `resin-<board>` release 1.1.1.
+* From v1.2
+    * The version of `meta-resin` is in the format is 2 numbers separated by a dot.
+    * The version of `resin-<board>` is in the format of 3 numbers separated by dots. The first two number are the meta-resin release while the last one is the `resin-<board>` release of the respective meta-resin release.
+    * Ex. For `meta-resin` release 1.2 there can be `resin-<board>` releases 1.2.X.
```

Comment:
I think we could also clarify what the version should look like when meta-resin switched from 1.2 to 1.3 As far as I remember, we agreed that this means the VERSION in resin-<board> will be 1.3, but thinking some more I think it's move intuitive that it is 1.3.0, and future board updates based on version 1.3 start from 1, Like: 1.3.1, 1.3.2, etc